### PR TITLE
Add setc2obas and setc2ohom for readability based on MC's comments

### PR DESCRIPTION
**Pull request by zwang123**
https://github.com/metamath/set.mm/pull/4201#issuecomment-2351380865

### CODE REVIEW

File: set.mm [hunk 1]
```diff
@@ -209475,6 +209476,37 @@ is always a subcategory (and it is full, meaning that all morphisms of
...
+      setc2obas $p |- ( (/) e. B /\ 1o e. B /\ 1o =/= (/) ) $=
+        ( c0 wcel c1o wne cpr 0ex prid1 cbs cfv c2o wceq wtru cvv 2oex eleqtrri
+        a1i setcbas mptru df2o3 3eqtr2i 1oex prid2 1n0 3pm3.2i ) EAFGAFGEHEEGIZ
+        AEGJKABLMZNUIDNUJOPBNQCNQFPRTUAUBUCUDZSGUIAEGUEUFUKSUGUH $.
+    $}
+
+    setc2ohom.h $e |- H = ( Hom ` C ) $.
+    $( ` ( SetCat `` 2o ) ` is a category (provable from ~ setccat and ~ 2oex )
+       that does not have pairwise disjoint hom-sets, proved by this theorem
+       conbined with ~ setc2obas .  Notably, the empty set ` (/) ` is
```
Comment (tirix) [inline set.mm:209496]: conbined > combined

File: set.mm [hunk 2]
```diff
@@ -209475,6 +209476,37 @@ is always a subcategory (and it is full, meaning that all morphisms of
...
+        a1i setcbas mptru df2o3 3eqtr2i 1oex prid2 1n0 3pm3.2i ) EAFGAFGEHEEGIZ
+        AEGJKABLMZNUIDNUJOPBNQCNQFPRTUAUBUCUDZSGUIAEGUEUFUKSUGUH $.
+    $}
+
+    setc2ohom.h $e |- H = ( Hom ` C ) $.
+    $( ` ( SetCat `` 2o ) ` is a category (provable from ~ setccat and ~ 2oex )
+       that does not have pairwise disjoint hom-sets, proved by this theorem
+       combined with ~ setc2obas .  Notably, the empty set ` (/) ` is
+       simultaneously an object ( ~ setc2obas ) , an identity morphism from
+       ` (/) ` to ` (/) ` , and a non-identity morphism in from ` (/) ` to
```
Comment (avekens) [inline set.mm:209498]: "in"?

File: set.mm (hunk 1 above)
Comment (zwang123) [reply set.mm:209496]: https://github.com/metamath/set.mm/pull/4239/commits/cdbf6c16113498b199ef0980dbb75ba452418793

File: set.mm (hunk 2 above)
Comment (zwang123) [reply set.mm:209498]: https://github.com/metamath/set.mm/pull/4239/commits/42b22d4d4b20168e2b342015c0cee7c685ab2524

**Review by icecream17 — APPROVED** [review]

**Review by avekens — APPROVED** [review]

**Review by tirix — APPROVED** [review]

**Pull request merged**